src/Aspire.Hosting.AWS/DynamoDB/DynamoDBLocalOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+using System.Diagnostics;
+
+namespace Aspire.Hosting.AWS.DynamoDB;
+
+/// <summary>
+/// Options that can be set for configuring the instance of DynamoDB local.
+/// </summary>
+[DebuggerDisplay("Type = {GetType().Name,nq}, Registry = {Registry}, Image = {Image}, Tag = {Tag}, LocalStorageDirectory = {LocalStorageDirectory}, SharedDb = {SharedDb}, DisableDynamoDBLocalTelemetry = {DisableDynamoDBLocalTelemetry}, DelayTransientStatuses = {DelayTransientStatuses}")]
+public class DynamoDBLocalOptions
+{
+    /// <summary>
+    /// The registry of the container image. THe default is public.ecr.aws.
+    /// </summary>
+    public string Registry { get; set; } = "public.ecr.aws";
+
+    /// <summary>
+    /// The container image to run for DynamoDB local. The default is aws-dynamodb-local/aws-dynamodb-local.
+    /// </summary>
+    public string Image { get; set; } = "aws-dynamodb-local/aws-dynamodb-local";
+
+    /// <summary>
+    /// The container image tag. The default is latest.
+    /// </summary>
+    public string Tag { get; set; } = "latest";
```

Comment:
Are you sure you want to use latest? It'll make people's experience unstable by default.

Reply:
I'll ping the DynamoDB team to see what they are comfortable with. They don't maintain a floating stable and latest version. So the alternative is hardcoding a specific version requiring a new release of the package every time there is a new version.